yarn.lock

```diff
@@ -0,0 +1,10901 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Why is this here? Do we have yarn now?

Reply:
I guess that's up to you. I've been using yarn. If not, then we'll need to make sure package-lock is updated.